### PR TITLE
Replacing PyMC3 plots w/ Arviz plots & sigma Param change [Part 1]

### PR DESCRIPTION
**Pull request by CloudChaoszero**
# Description
The following is a large PR breakdown of PR #16.

Replace PyMC3 dependent plots with arviz plots in case studies & examples.
* The issue is a 3-part step for the following:
  * Initiative via Issue https://github.com/pymc-devs/pymc3/issues/4371 (Leave open)

Replace parameter `sd` with `sigma` (e.g. some examples have `pm.Normal(...sd=...)`
* Closes issue #2

### CODE REVIEW

File: examples/case_studies/disaster_model.py [hunk 1]
```diff
@@ -152,4 +152,4 @@
     start = {"early_mean": 2.0, "late_mean": 3.0}
 
     tr = pm.sample(1000, tune=500, start=start)
-    pm.traceplot(tr)
+    pm.plot_trace(tr)
```
Comment (MarcoGorelli) [inline examples/case_studies/disaster_model.py:155]: should this also be `az.plot_trace`?

File: examples/pymc3_howto/LKJ_correlation.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def run(n=1000):
         n = 50
     with model:
         trace = pm.sample(n)
-    pm.traceplot(
+    az.plot_trace(
         trace, varnames=["mu", "r"], lines={"mu": mu_r, "r": corr_r[np.triu_indices(n_var, k=1)]}
```
Comment (OriolAbril) [inline examples/pymc3_howto/LKJ_correlation.py:57]: I am quite sure arviz does not accept `varnames`, it should be `var_names`

File: examples/pymc3_howto/rankdata_ordered.py [hunk 3]
```diff
@@ -46,14 +46,14 @@ def run(n=1500):
     with m:
         trace = pm.sample(n)
 
-    pm.traceplot(trace, varnames=["mu_hat"])
+    az.plot_trace(trace, varnames=["mu_hat"])
```
Comment (OriolAbril) [inline examples/pymc3_howto/rankdata_ordered.py:49]: `var_names`

File: examples/pymc3_howto/rankdata_ordered.py [hunk 4]
```diff
@@ -46,14 +46,14 @@ def run(n=1500):
...
 
     print("Example observed data: ")
     print(y[:30, :].T)
     print("The true ranking is: ")
     print(yreal.flatten())
     print("The Latent mean is: ")
-    latentmu = np.hstack(([0], pm.summary(trace, varnames=["mu_hat"])["mean"].values))
+    latentmu = np.hstack(([0], az.summary(trace, varnames=["mu_hat"])["mean"].values))
```
Comment (OriolAbril) [inline examples/pymc3_howto/rankdata_ordered.py:56]: `var_names`

**Review by OriolAbril — COMMENTED** [review]
Thanks! This is great!

Small roadmap question, what is the plan on using `return_inferencedata=True` regarding examples?

File: examples/pymc3_howto/LKJ_correlation.py (hunk 2 above)
Comment (CloudChaoszero) [reply examples/pymc3_howto/LKJ_correlation.py:57]: Ah, correct! Thanks for that. I updated to arviz `0.11`, and got this error

```
y:212: UserWarning: A valid var_name should be provided, found {'m'} expected from {'mu', 'r'}
  warnings.warn(
Traceback (most recent call last):
  File "LKJ_correlation.py", line 62, in <module>
    run()
  File "LKJ_correlation.py", line 56, in run
    az.plot_trace(
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/traceplot.py", line 238, in plot_trace
    axes = plot(**trace_plot_args)
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/backends/matplotlib/traceplot.py", line 384, in plot_trace
    for _, _, vlines in (j for j in lines if j[0] == var_name and j[1] == selection):
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/backends/matplotlib/traceplot.py", line 384, in <genexpr>
    for _, _, vlines in (j for j in lines if j[0] == var_name and j[1] == selection):
IndexError: string index out of range
```
(This error looks related https://github.com/arviz-devs/arviz/issues/988) 

Any thoughts why this happened?

File: examples/case_studies/disaster_model.py (hunk 1 above)
Comment (CloudChaoszero) [reply examples/case_studies/disaster_model.py]: Acknowledged, thanks for that!

**Comment by CloudChaoszero** [comment]
> Small roadmap question, what is the plan on using `return_inferencedata=True` regarding examples?

Yeah @OriolAbril ! I was hoping to get these ~9 PRs merged into the repo, and the proceed into that.
These PRs just focus on 
* sigma param
* Plot dependencies
However, I am still have some delays on my side 😅

**Review by OriolAbril — APPROVED** [review]
That timeline sounds great @CloudChaoszero! Thanks!

File: examples/pymc3_howto/LKJ_correlation.py (hunk 2 above)
Comment (CloudChaoszero) [reply examples/pymc3_howto/LKJ_correlation.py]: I think I found out why now. It's due to the `lines` param.

I removed it to make the `az.plot_trac()` work.

Comment (OriolAbril) [reply examples/pymc3_howto/LKJ_correlation.py]: Try

    lines=(("mu", {}, mu_r), ("r", {}, corr_r[np.triu_indices(n_var, k=1)]))

**Pull request merged**